exports.js

```diff
@@ -989,7 +989,8 @@ module.exports = {
         'eventHubMinimumTLSversion'     : require(__dirname + '/plugins/azure/eventhub/eventHubMinimumTLSversion.js'),
 
         'accessLogsEnabled'             : require(__dirname + '/plugins/azure/frontdoor/accessLogsEnabled.js'),
-        'frontDoorMinimumTlsVersion'    : require(__dirname + '/plugins/azure/frontdoor/frontDoorMinimumTlsVersion.js')
+        'frontDoorMinimumTlsVersion': require(__dirname + '/plugins/azure/frontdoor/frontDoorMinimumTlsVersion.js'),
+        'dbDataDiscoveryClassification': require(__dirname + '/plugins/azure/sqldatabases/dbDataDiscoveryClassification.js'),
```

Comment:
```suggestion
        'dbDataDiscoveryClassification': require(__dirname + '/plugins/azure/sqldatabases/dbDataDiscoveryClassification.js'),
```
move it under other sqldatabases plugins